fixtures/seed-cf.ts

```diff
@@ -0,0 +1,72 @@
+import getImageSizes from './image-sizes'
+import getGlobals from './globals'
+
+interface SeedCF {
+  fl_content: any[]
+  fl_environments: any[]
+  fl_files: any[]
+  fl_folders: any[]
+  fl_locales: any[]
+  fl_navigation: any[]
+  fl_permissions: any[]
+  fl_schemas: any[]
+  fl_settings: any[]
+  fl_users: any[]
+}
+
+export default (): SeedCF => ({
```

Comment:
@jordanch & @gitdubz This method handles seeding the CF database with the base data required for our unit tests. So far it only adds some of the settings and users that is required for the current tests - I'll expand it as the tests are added.

It is important to note the structure if you want to add more content:

- there is a property for each collection that should be generated with an array value
- each array should contain objects for each document that should be generated
- the documents should have a `docId` and a `doc` that is the Firestore document ID and document body respectively
- if you need a Firestore Document Reference, make the field's value an object with a `type: 'ref'` and `val` property. The `val` prop needs to be the string path to the reference you want. See `fl_settings > defaultPermissionsGroup` as an example below.